classes/class-wc-connect-payment-methods-store.php

```diff
@@ -104,11 +104,11 @@ protected function get_payment_methods_from_response_body( $response_body ) {
 				return new WP_Error( 'payment_methods_type', 'Expected but did not receive array for payment_methods.' );
 			}
 
-			foreach ( (array) $payment_methods as $payment_method ) {
+			foreach ( $payment_methods as $payment_method ) {
 				$required_keys = array( 'payment_method_id', 'name', 'card_type', 'card_digits', 'expiry' );
-				foreach ( (array) $required_keys as $required_key ) {
-					if ( ! array_key_exists( $required_key, $payment_method ) ) {
-						return new WP_Error( 'payment_methods_key_missing', 'Payment method array element is missing a required key' );
+				foreach ( $required_keys as $required_key ) {
+					if ( ! property_exists( $payment_method, $required_key ) ) {
```

Comment:
`property_exists` returns `true` even if the value is `null`. Is this okay? Otherwise we might need `isset`

Reply:
`array_key_exists` behaves the same way so I'm sure it's ok. I'm wonder why we are even doing this check to begin with but I'm just fixing the notice.